contrib/production/kcp-comer/certificate-etcd.yaml

```diff
@@ -0,0 +1,144 @@
+---
+# Root etcd CA certificate for all etcd components across all kcp deployment.
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: etcd-ca
+  namespace: kcp-comer
+spec:
+  secretName: etcd-ca-tls
+  isCA: true
+  commonName: etcd-ca
+  issuerRef:
+    name: etcd-ca
+    kind: ClusterIssuer
+---
+# etcd Peer CA certificate (for peer communication)
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: etcd-peer-ca
+  namespace: kcp-comer
+spec:
+  secretName: etcd-peer-ca-tls
+  isCA: true
+  commonName: etcd-peer-ca
+  issuerRef:
+    name: etcd-ca
+    kind: ClusterIssuer
+---
+# etcd Peer CA issuer
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: etcd-peer-ca
+  namespace: kcp-comer
+spec:
+  ca:
+    secretName: etcd-peer-ca-tls
+---
+# etcd server certificate (for client connections)
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: etcd-server-tls
+  namespace: kcp-comer
+spec:
+  secretName: etcd-server-tls
+  commonName: etcd-server
+  dnsNames:
+  - root-local
+  - root-client
+  - root-client.kcp-comer
+  - root-client.kcp-comer.svc
+  - root-client.kcp-comer.svc.cluster.local
+  - "*.root-peer"
+  - "*.root-peer.kcp-comer"
+  - "*.root-peer.kcp-comer.svc"
+  - "*.root-peer.kcp-comer.svc.cluster.local"
+  - alpha-local
+  - alpha-client
+  - alpha-client.kcp-comer
+  - alpha-client.kcp-comer.svc
+  - alpha-client.kcp-comer.svc.cluster.local
+  - "*.alpha-peer"
+  - "*.alpha-peer.kcp-comer"
+  - "*.alpha-peer.kcp-comer.svc"
+  - "*.alpha-peer.kcp-comer.svc.cluster.local"
+  issuerRef:
+    name: etcd-ca
```

Comment:
Why is this not using `etcd-ca` from the `kcp-comer` namespace, similar to `etcd-tls-ca`?

Reply:
This is all etcd operator specific. This is replicated in each "namespace"

Reply:
Might be miskate. Need to test before I change it. TBH re-testing this means rebuilding everything. So will take a note and once I do next rebuild - will check.